### DISCUSSION TITLE
Tree path name as label in tree select

### DISCUSSION THREAD

**Discussion by taruntadikonda** [primefaces/community#249] [PrimeNG] [answered]
![image](https://github.com/primefaces/community/assets/25893079/7e04507f-a462-4124-86dc-8d914a4de74e)

From the above image the selected node is Expenseses.doc and path to it is Documents > Work >  Expenseses.doc. Is it possible to show 
Documents.Work.Expenseses.doc as the label in dropdown. Any parameter in the TreeSelect interface that can be assigned to change the
label .

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example to identify the issue clearly?

**Reply by taruntadikonda** [reply]
It is not a issue. I wanted to know whether it is possible through some configuration.

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

Sorry for the misunderstanding, there is a value template for the customization of selected values. Here is a basic example of value template https://stackblitz.com/run?file=src%2Fapp%2Fdemo%2Ftree-select-checkbox-demo.html
